MonoidalCategories/gap/BraidedMonoidalCategoriesTest.gd

```diff
@@ -19,6 +19,10 @@
 #! with the parameters given above and
 #! compared to the equivalent computation in
 #! the opposite category of $cat$.
-#! Pass the option 'verbose := true' to output more information.
+#! Pass the options 
+#! * 'verbose := true' to output more information.
+#! * 'only_primitive_operations := true' to only use 
+#!    dual operations for operations primitively installed in $cat$.
```

Comment:
For me the phrasing sounds a bit misleading because I read "use" as "test" in this context. Suggestion:
```suggestion
#! * 'only_primitive_operations := true' to only primitively install 
#!    dual operations for operations primitively installed in $cat$.
```

And I think it would be good to also mention the downsides, i.e. that fewer dual_pre/postprocessor_funcs might get tested.

Reply:
> For me the phrasing sounds a bit misleading because I read "use" as "test" in this context. [...] And I think it would be good to also mention the downsides, i.e. that fewer dual_pre/postprocessor_funcs might get tested.

Yes, I wasn't satisfied with my wording as well. :/ I also don't like how complicated that sentence reads, maybe we can briefly talk about it tomorrow.